styleguide/gulpfile.js

```diff
@@ -107,9 +108,9 @@ gulp.task('minifyIcons', function() {
 });
 
 // Based on https://github.com/filamentgroup/gulpicon#usage
-var iconFiles = glob.sync("source/assets/icons/svg/*.svg");
-var iconConfig = require("./source/assets/icons/config.js");
-iconConfig.dest = "public/assets/icons/";
+var iconFiles = glob.sync('source/assets/icons/svg/*.svg');
```

Comment:
These lines were out of sync with the single-quote standard we use everywhere else which is why I left this correction.